CHANGELOG.md

```diff
@@ -2,7 +2,11 @@
 
 Breaking changes:
 
+- [#1645](https://github.com/rails-api/active_model_serializers/pull/1645) Expanded KeyTransform into Transform. Changed :dashed transform to :dash. (@remear)
```

Comment:
Sorry, I didn't read carefully. I mean from `KeyTransform into Transform`. though `dashed to dash` is a good question, too.

How does that

> To explain what actually caused the breakage.